tools/Microsoft.TemplateSearch.TemplateDiscovery/PackChecking/FilteredPackageInfo.cs

```diff
@@ -6,6 +6,7 @@
 
 namespace Microsoft.TemplateSearch.TemplateDiscovery.PackChecking
 {
+    [System.Diagnostics.DebuggerDisplay("{Name}@{Version} - {Reason}")]
```

Comment:
This workflow deals with large lists of these items, having a debugger visualization is very handy.